src/modules/translation/languages/zh.json

```diff
@@ -1021,6 +1022,7 @@
     "beneficiary": "受益人",
     "supply": "供应",
     "collection": "采集",
+    "edit_urn": "编辑 URN",
```

Comment:
Isn't `copy_urn` missing?

Reply:
it was, I had a mixup with different branches it seems